src/test/java/pages/ProfilePage.java

```diff
@@ -0,0 +1,63 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import java.util.UUID;
+
+public class ProfilePage extends BasePage {
+
+    // Constructor for ProfilePage Class invoking super class constructor
+    public ProfilePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    // Locators for ProfilePage class
+    By profileName = By.cssSelector("span.name");
```

Comment:
Hi,
I have checked your homework 22 about Page Object Model (POM). You're doing great! Keep up the good work!
Best wishes from the HW team (Khanh, khanh@testpro.io)

Test script PASSED
![image](https://github.com/user-attachments/assets/a4876245-4d79-443a-a081-852feac0008b)

By the way, I would suggest  you can use attribute selectors to define profileName to more clearly
ex: `By profileName = By.cssSelector("a[class='view-profile']");`